### PR TITLE
Split AzureSDK.podspec and add AzureCommunicationCalling.podspec

### PR DESCRIPTION
**Pull request by raosanat**
Split AzureSDK.podspec into individual podspecs and add new podspec for AzureCommunicationCalling

### CODE REVIEW

File: AzureCommunicationCalling.podspec [hunk 1]
```diff
@@ -25,40 +25,27 @@
...
+  
+    s.ios.deployment_target = '12.0'
+    s.ios.vendored_frameworks = "AzureCommunicationCalling.framework"
+
+    s.swift_version = '5.0'
+  
+    s.source = { :http => 'https://github.com/Azure/Communication/releases/download/v1.0.0-beta.4/azurecommunicationcalling.framework-1.0.0-beta.4.zip' }
+    s.source_files = 'Headers/*.h'
+    s.public_header_files = "Headers/*.h"
+    s.dependency 'AzureCore', '~> 1.0.0-beta.2'
```
Comment (bsiegel) [inline AzureCommunicationCalling.podspec:48]: I don't believe Calling depends on AzureCore any longer, double check but we should be able to remove this.

File: EULA.txt [hunk 2]
```diff
@@ -0,0 +1,79 @@
+MICROSOFT SOFTWARE LICENSE TERMS
```
Comment (bsiegel) [inline EULA.txt:1]: Can we move this into e.g. `sdk/communiation/AzureCommunicationCalling/` ? I think seeing this in the root of the repo might concern users. I'm fine if this EULA is the only file in that directory.

File: AzureCommunicationCalling.podspec (hunk 1 above)
Comment (raosanat) [reply AzureCommunicationCalling.podspec:48]: Dependency needs to be still there for older releases we have done.

Comment (bsiegel) [reply AzureCommunicationCalling.podspec:48]: If the podspec is starting at 1.0.0-beta.4 and 1.0.0-beta.4 doesn't require AzureCore I don't see any reason it couldn't be removed.

File: EULA.txt (hunk 2 above)
Comment (raosanat) [reply EULA.txt]: Moved

File: AzureCommunicationCalling.podspec (hunk 1 above)
Comment (raosanat) [reply AzureCommunicationCalling.podspec:48]: Yes now i get it, AzureCommunication already specifes deps on AzureCore , so yes we dont directly dont depend on AzureCore

**Review by bsiegel — APPROVED** [review]
:tada: Looks good, all specs passed validation and got pushed to my Specs repo https://github.com/bsiegel/Specs

:shipit:

**Pull request merged**